### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by nden**

### CODE REVIEW

File: tutorial/install/environment.yml [hunk 1]
```diff
@@ -17,4 +17,4 @@ dependencies:
       - astropy >= 5.3
       - gwcs >= 0.20
       - stpipe
```
Comment (braingram) [inline tutorial/install/environment.yml]:
```suggestion
```
Do we need `stpipe`?

File: tutorial/install/environment.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ dependencies:
   - numpy >= 1.20
   - packaging
   - pip
-  - python >= 3.9
+  - python == 3.11
   - pyyaml
   - pip:
       - asdf_standard
```
Comment (braingram) [inline tutorial/install/environment.yml]:
```suggestion
```
I don't think we use the standard directly.

File: tutorial/data/download.py~
Comment (braingram) [inline tutorial/data/download.py~:1]: Temporary file?

File: tutorial/.DS_Store
Comment (braingram) [inline tutorial/.DS_Store:1]: Included before the `.gitignore`?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,168 @@
...
+Making sure to activate the `conda` environment (`conda activate adass-asdf`), run the following to
+download the data files used by these notebooks:
+
+```shell
+python data/download.py
+```
+
+## 7. Starting Jupyter Notebook
+
+Making sure your terminal is in the `roman-data-workshop` directory (you can use `pwd` to check), you can then start the
```
Comment (braingram) [inline README.md]:
```suggestion
Making sure your terminal is in the `addf-adass2024` directory (you can use `pwd` to check), you can then start the
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,168 @@
...
+
+Open a terminal window using the appropriate one for your operating system.
+
+Now navigate to this directory in the terminal:
+
+```shell
+cd asdf-adass2024
+```
+
+And finally, on any platform, to install and activate the `roman-data-workshop-env` environment, type:
```
Comment (braingram) [inline README.md]:
```suggestion
And finally, on any platform, to install and activate the `adass-asdf` environment, type:
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,168 @@
...
-This is a work in progress.
\ No newline at end of file
+To run all the workshop notebooks on your own computer, be sure your machine is configured with the packages in
+the
+[installation check file](https://github.com/asdf-format/asdf-adass2024/blob/main/tutorial/install/verify_install.py). These packages
+are the ones we use to verify that the notebooks are working as expected.
+
+These instructions assume the installation is done on a *nix type of operating system and describe setup using `git` and `Miniconda`. It is not strictly necessary to use either of these.There are pointers for setting this up on Windows, however Windows is not fully supported.
+
+## 1. Install Miniconda (if needed)
```
Comment (braingram) [inline README.md:12]: I must admit that I very rarely use conda/miniconda/mamba. It looks good to me but I don't know the "gotchas" for conda.

**Review by braingram — APPROVED** [review]
Thanks! A few comments/suggestions

File: tutorial/install/environment.yml (hunk 1 above)
Comment (nden) [reply tutorial/install/environment.yml]: no, that was tied to romancal

File: README.md (hunk 5 above)
Comment (nden) [reply README.md:12]: I think that's what most astronomers are used to, we'll find out.

**Pull request merged**